jwql/website/apps/jwql/monitor_pages/monitor_cosmic_rays_bokeh.py

```diff
@@ -0,0 +1,165 @@
+"""This module contains code for the cosmic ray monitor Bokeh plots.
+
+Authors
+-------
+
+    - Mike Engesser
+Use
+---
+
+    This module is intended to be imported and use as such:
+    ::
+
+        from jwql.website.apps.jwql import monitor_pages
+        monitor_template = monitor_pages.CosmicRayMonitor()
+"""
+
+import datetime
+import os
+
+from astropy.io import fits
```

Comment:
Looks like the fits import can be removed. I don't think it's used in here.

Reply:
@bhilbert4 are we going to implement these changes?

Reply:
This has already been removed.